dashboard/src/shared/types.ts

```diff
@@ -3,6 +3,7 @@ import { IServiceCatalogState } from "../reducers/catalog";
 import { IFunctionState } from "../reducers/functions";
 import { INamespaceState } from "../reducers/namespace";
 import { IAppRepositoryState } from "../reducers/repos";
+import { IConfig } from "./Config";
```

Comment:
What's the reason for the I prefix?

Reply:
enforced by tslint https://palantir.github.io/tslint/rules/interface-name/